Samples/WindowPlacement/cpp/SaveRestoreSample/SaveRestoreSample.cpp

```diff
@@ -0,0 +1,420 @@
+
+#define USE_VIRTUAL_DESKTOP_APIS
```

Comment:
I dislike that these are not enabled by default. Can we choose opt-out instead? `USE_VIRTUAL_DESKTOP_APIS` and `USE_WINDOW_ACTION_APIS`

In private discussion, we're ok making WINDOW_ACTION default, but Virtual Desktops require COM. Let's at least document this opt-in.